AUTHORS

```diff
@@ -15,1315 +15,1314 @@
 # This should contain the preferred addresses.  Alternate addresses are in
 # Porting/checkAUTHORS.pl.
 #
-# After editing, please run: make test_porting
+# updated_by: Porting/checkAUTHORS.pl --update --from=v5.x.x
 -- 
-A. C. Yardley			<yardley@tanet.net>
-A. Sinan Unur			<nanis@cpan.org>
-Aaron B. Dossett		<aaron@iglou.com>
-Aaron Crane			<arc@cpan.org>
-Aaron J. Mackey			<ajm6q@virginia.edu>
-Aaron Priven			<aaron@priven.com>
-Aaron Trevena			<aaaron.trevena@gmail.com>
-Abe Timmerman			<abe@ztreet.demon.nl>
-Abhijit Menon-Sen		<ams@toroid.org>
-Abigail				<abigail@abigail.be>
-Abir Viqar			<abiviq@hushmail.com>
-Achim Bohnet			<ach@mpe.mpg.de>
-Achim Gratz			<achim.gratz@stromeko.de>
-Adam Flott			<adam@npjh.com>
-Adam Kennedy			<adam@ali.as>
-Adam Krolnik			<adamk@gypsy.cyrix.com>
-Adam Milner			<carmiac@nmt.edu>
-Adam Russell			<arussell@cs.uml.edu>
+A. C. Yardley                  <yardley@tanet.net>
+A. Sinan Unur                  <nanis@cpan.org>
+Aaron B. Dossett               <aaron@iglou.com>
+Aaron Crane                    <arc@cpan.org>
+Aaron J. Mackey                <ajm6q@virginia.edu>
```

Comment:
Why not:

- preserve existing valid entries, or
- use tabs (which preserves most existing valid entries)

to reduce the noise in this change?

Reply:
I see, I was trying to honor the preferences set in the DATA section... 
I changed several of them to reduce the noise.

Reply:
most of the noise come from sorting the authors and lowercase on some emails
when reviewing with `-w` the noise is very limited

Reply:
I can see altering the DATA section to reduce the changes...
and match the existing email in AUTHORS so it will reduce complaints...